src/state/assetsSelectors.ts

```diff
@@ -16,3 +19,14 @@ export const getCurrentMarketAssetData = (state: RootState) => {
   const { assetId } = getCurrentMarketData(state) ?? {};
   return assetId ? getAssets(state)?.[assetId] : undefined;
 };
+
+/**
+ * @returns Asset data of specified AssetId
+ */
+export const getAssetData = (state: RootState, assetId: Nullable<string>) =>
```

Comment:
I think it's better if this isn't a selector. I imagine the memoization is getting churned constantly anyway

Reply:
yeah definitely doesn't need to be a selecto

Reply:
👍